Library/ViewModels/CommentCellViewModel.swift

```diff
@@ -124,16 +127,19 @@ public final class CommentCellViewModel:
       .map(userIsBackingCreatorOrCollaborator)
       .negate()
 
+    let isReply = comment.map { $0.isReply }
+
     // If the user is either logged out, not backing or the flag is disabled, hide replyButton.
     self.replyButtonIsHidden = Signal.combineLatest(isLoggedOut, isNotABackerCreatorOrCollaborator)
       .map(replyButtonHidden)
 
     // If both the replyButton and flagButton should be hidden, the entire stackview will be hidden too.
     self.bottomRowStackViewIsHidden = Signal.combineLatest(
       self.flagButtonIsHidden.signal,
-      self.replyButtonIsHidden.signal
-    ).map { flagButtonIsHidden, replyButtonIsHidden in
-      flagButtonIsHidden && replyButtonIsHidden
+      self.replyButtonIsHidden.signal,
+      isReply
+    ).map { flagButtonIsHidden, replyButtonIsHidden, isReply in
+      (flagButtonIsHidden && replyButtonIsHidden) || isReply
```

Comment:
I'm assuming you added `|| isReply` because we don't support flagging and replying to replies?

Reply:
That's correct